akka-stream/src/main/scala/akka/stream/impl/fusing/StreamOfStreams.scala

```diff
@@ -427,11 +427,11 @@ import scala.util.control.NonFatal
           tryCompleteHandler()
         }
 
-        override def onDownstreamFinish(): Unit = {
+        override def onDownstreamFinish(cause: Throwable): Unit = {
```

Comment:
I guess it means here that we can only report the last cancellation cause, but I guess that's good enough for now.

Reply:
Yeah, didn't think it was worth it to collect the failures